crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -95,9 +99,35 @@ def deactivate
     end
   end
 
+  #
+  # Activate all Repositories
+  #
+  # Provides the restful api call for
+  # Activate all Repositories   /utils/repositories/activate_all   POST  Creates Repository DataBagItem
+  def activate_all
+    respond_to do |format|
+      ProvisionerService.new(logger).enable_all_repositories
```

Comment:
This should be outside of the respond_to block. Maybe even for the other actions as well.